src/devices/Ad7193/Ad7193.cs

```diff
@@ -0,0 +1,855 @@
+                SetRegisterValue(Register.Configuration, _registerCache[(byte)Register.Configuration]);
+            }
+        }
+
+        /// <summary>
+        /// The value of the current Status register
+        /// </summary>
+        public uint Status
+        {
+            get
+            {
+                _registerCache[(byte)Register.Status] = GetRegisterValue(Register.Status);
+                return _registerCache[(byte)Register.Status];
+            }
+        }
+
+        /// <summary>
+        /// The value of the current Mode register
+        /// </summary>
+        public uint Mode
+        {
+            get
+            {
+                _registerCache[(byte)Register.Mode] = GetRegisterValue(Register.Mode);
+                return _registerCache[(byte)Register.Mode];
+            }
+        }
+
+        /// <summary>
+        /// The value of the current Configuration register
+        /// </summary>
+        public uint Config
+        {
+            get
+            {
+                _registerCache[(byte)Register.Configuration] = GetRegisterValue(Register.Configuration);
+                return _registerCache[(byte)Register.Configuration];
+            }
+        }
+
+        /// <summary>
+        /// Power-down mode. In power-down mode, all AD7193 circuitry, except the bridge power-down switch, is powered down. The bridge power-down switch remains active because the user may need to power up the sensor prior to powering up the AD7193 for settling reasons. The external crystal, if selected, remains active.
+        /// </summary>
+        public bool IsPowerDown
+        {
+            get
+            {
+                GetRegisterValue(Register.Mode);
+                return (GetRegisterBits(Register.Mode, BitMask.ModeMD) == 0b011);
+            }
+        }
+
+        /// <summary>
+        /// Ready bit for the ADC. This bit is set when data is written to the ADC data register.
+        /// </summary>
+        public bool IsReady
+        {
+            get
+            {
+                GetRegisterValue(Register.Status);
+                return (GetRegisterBits(Register.Mode, BitMask.StatusRDY) != 0b1);
+            }
+        }
+
+        /// <summary>
+        /// True is the ADC has errors
+        /// </summary>
+        public bool HasErrors
+        {
+            get
+            {
+                uint status = GetRegisterValue(Register.Status);
+
+                return (status & 0b0100_0000) == 0b0100_0000;
+            }
+        }
+
+        private bool _continuousRead = false;
+
+        /// <summary>
+        /// Is the ADC in continuous conversion mode
+        /// </summary>
+        public bool ContinuousRead
+        {
+            get
+            {
+                return _continuousRead;
+            }
+
+            set
+            {
+                if (value)
```

Comment:
This could be collapsed to:

```csharp
SetRegisterValue(Register.Communications, value ? 0b0101_1100 : 0b0101_1000);
```

Reply:
This is just the question of preference, I think, but true. I changed it.